libs/components/packages/src/schematics/migrations/update-13/icon-name/icon-name.schematic.spec.ts

```diff
@@ -122,6 +122,18 @@ describe('icons-name.schematic', () => {
       <sky-icon iconName="bb-diamond" variant="solid"></sky-icon>
       `,
       },
+      {
+        invalidTemplate: `
+      <sky-icon iconName="add" fixedWidth="true"/>
+      <sky-icon iconName="add" [fixedWidth]="true"/>
+      <sky-icon iconName="add" fixedWidth/>
```

Comment:
This is a good test - but should we also have a case that checks converting icon and removing fixed width at the same time? 

And maybe one where `icon` can't be converted but `fixedWidth` is still removed?